_targets.R

```diff
@@ -24,19 +26,29 @@ major_ion_names = c("Chloride","Sodium")
 # Define hydrologic event types in harmonized WQP data to exclude
 omit_wqp_events <- c("Spill","Volcanic action")
 
-# Define USGS specific conductance parameter codes
-SpC_pcodes <- c("00095","90095","00094","90096") 
+# Define all USGS specific conductance parameter codes
+pcodes_select <- c("00095","90095","00094","90096") 
 
-# Define minor HUCs (hydrologic unit codes) that make up the DRB (HUC04 = 0204)to use in calls to dataRetrieval functions
+# Define USGS specific conductance parameter code to use for continuous data
+parameter <- "00095"
+
+# Define minor HUCs (hydrologic unit codes) that make up the DRB to use in calls to dataRetrieval functions
+# Lower Delaware: 0204 subregion (for now, exclude New Jersey Coastal (https://water.usgs.gov/GIS/huc_name.html)
```

Comment:
did you add this note to the markdown? Are the NJ coastal HUCs the 2 deleted?

Reply:
I originally included the two NJ Coastal HUCs because they're within the Delaware subregion but have since omitted them - the DRB harmonized WQP dataset does not include those HUCs and neither does the PRMS dataset from the DRB temperature work. Let me know if you think otherwise. Either way, I'll add a note to the markdown doc.

Reply:
Okay, thanks! Great reasons to omit them, then